configs/det/dbnet/db++_r50_icdar15_910B.yaml

```diff
@@ -0,0 +1,160 @@
+system:
+  mode: 0 # 0 for graph mode, 1 for pynative mode in MindSpore
+  distribute: False
+  amp_level: 'O0'
+  seed: 42
+  log_interval: 10
+  val_while_train: True
+  val_start_epoch: 800
+  val_interval: 1
+  drop_overflow_update: True
+
+model:
+  type: det
+  transform: null
+  backbone:
+    name: det_resnet50
+    pretrained: False
+  neck:
+    name: DBFPN
+    out_channels: 256
+    bias: False
+    use_asf: True             # Adaptive Scale Fusion
+    channel_attention: True   # Use channel attention in ASF
+  head:
+    name: DBHead
+    k: 50
+    bias: False
+    adaptive: True
+  pretrained: https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_synthtext-40655acb.ckpt
+
+postprocess:
+  name: DBPostprocess
+  box_type: quad   # whether to output a polygon or a box
+  binary_thresh: 0.3      # binarization threshold
+  box_thresh: 0.6         # box score threshold
+  max_candidates: 1000
+  expand_ratio: 1.5       # coefficient for expanding predictions
+
+metric:
+  name: DetMetric
+  main_indicator: f-score
+
+loss:
+  name: DBLoss
+  eps: 1.0e-6
+  l1_scale: 10
+  bce_scale: 5
+  bce_replace: bceloss
+
+scheduler:
+  scheduler: polynomial_decay
+  lr: 0.007
+  num_epochs: 2000
+  decay_rate: 0.9
+  warmup_epochs: 3
+
+optimizer:
+  opt: momentum
+  filter_bias_and_bn: false
+  momentum: 0.9
+  weight_decay: 1.0e-4
+
+# only used for mixed precision training
+loss_scaler:
+  type: dynamic
+  loss_scale: 1024 
+  scale_factor: 2
+  scale_window: 1000
+
+train:
+  ema: True
+  ckpt_save_dir: './tmp_det'
+  dataset_sink_mode: True
+  dataset:
+    type: DetDataset
+    dataset_root: /home/hyx/datasets
```

Comment:
keep the dataset path generic as in other configs: `/data/ocr_datasets`.

Reply:
done